src/fn_call.rs

```diff
@@ -596,18 +596,59 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
                 this.write_scalar(Scalar::from_uint(key, dest.layout.size), dest)?;
             }
             "TlsGetValue" => {
-                let key = this.read_scalar(args[0])?.to_bits(args[0].layout.size)?;
+                let key = this.read_scalar(args[0])?.to_usize(this)? as u128;
```

Comment:
`DWORD` is `32` bit on all systems: https://msdn.microsoft.com/en-us/library/cc230318.aspx

Reply:
I thought I got errors when I used `u32`...